### DISCUSSION TITLE
Enabling custom mode on battery on 2022 Legion Slim 7 acting weird

### DISCUSSION THREAD

**Discussion by pinakiitkgp** [BartoszCichecki/LenovoLegionToolkit#814] [Q&A]
Anyone tried to enable custom mode on battery using the additional argument command described in FAQ? I tried to do it but something weird happened. It accepted the change but the fans spin very fast and loudly before quitting the software and it didn't work again. Is there any other way to implement it or am I doing something wrong?

What I did to enable: I went to the local data legion tool kit folder from file explorer and then created a new text file there and added the line --allow-all-power-modes-on-battery in the blank text file and saved it. Then I quit the folder and restarted toolkit.

**Comment by BartoszCichecki** [comment]
I hope you read the notice in the readme? It is not surprising that something is wrong, because on most laptops something is wrong if you force custom or performance mode on battery.

**Comment by pinakiitkgp** [comment]
Ya, I read it. I thought it acted weird because maybe I messed up in following the procedure properly. Sorry, I am not very good at these. Could you please confirm if I infact followed the procedure correctly you mentioned in FAQ?

**Reply by BartoszCichecki** [reply]
Yes. Looks like you did it correctly. But as written in the readme it is not guaranteed to works, that's why it's not enabled by default.
